public/src/app/user/services/user.service.ts

```diff
@@ -143,7 +143,7 @@ export class OSKUserService implements OnDestroy {
               email: data['email']!,
               publicProfile: data['publicProfile']!,
               privateProfile: data['privateProfile']!,
-              creationDate: (data['creationDate'] as Timestamp).toDate() ?? Date.now(),
```

Comment:
Hi,

This change should be reverted. If the field creationDate is filled, it should be used instead of the actual date.